backward.hpp

```diff
@@ -3847,6 +3847,9 @@ class Printer {
 
 #if defined(BACKWARD_SYSTEM_LINUX) || defined(BACKWARD_SYSTEM_DARWIN)
 
+const size_t stack_size = 8 * 1024 * 1024;
+static char *alt_stack = static_cast<char *>(malloc(stack_size));
```

Comment:
yes but... this allocates RAM out of the control of the user. The intend of the SignalHandling class is to be used only if you want it. If you don't you shouldn't be paying anything for it.